torch_geometric/graphgym/model_builder.py

```diff
@@ -17,14 +25,37 @@
 
 
 class GraphGymModule(LightningModule):
-    def __init__(self, dim_in, dim_out, cfg):
+    def __init__(self, dim_in, dim_out, cfg: "CfgNode"):
         super().__init__()
+        self.cfg = cfg
         self.model = network_dict[cfg.model.type](dim_in=dim_in,
                                                   dim_out=dim_out)
 
     def forward(self, *args, **kwargs):
         return self.model(*args, **kwargs)
 
+    def configure_optimizers(self) -> Tuple[Any, Any]:
+        optimizer = create_optimizer(self.model.parameters(), self.cfg.optim)
+        scheduler = create_scheduler(optimizer, self.cfg.optim)
+        return [optimizer], [scheduler]
+
+    def _shared_step(self, batch, split: str) -> Dict:
+        batch.split = split
+        pred, true = self.forward(batch)
+        loss, pred_score = compute_loss(pred, true)
+        step_end_time = time.time()
+        return dict(loss=loss, true=true, pred=pred, pred_score=pred_score,
```

Comment:
Not sure I understand the output here. What does `true=true` mean? And why do we only set `stop_end_time`?

Reply:
this is for logging `time_used` in the `logger.update(...)` method. And with the `LoggerCallback` I have created the `*_epoch_start_time` at the start of each [epoch like here](https://github.com/aniketmaurya/pytorch_geometric/blob/implement_module/torch_geometric/graphgym/logger.py#L288)

Reply:
regarding `true=true`, I have saved the values to access it later via the Logger callback.

```python
print(dict(true=true))
# {"true": [SOME TENSOR VALUE]}
```